expertsystem/io/__init__.py

```diff
@@ -9,11 +9,14 @@
 
 from expertsystem.data import ParticleCollection
 
+from . import _pdg
 from . import xml
 from . import yaml
 
 
-def load_particle_collection(filename: str) -> ParticleCollection:
+def load_particle_collection(filename: str = "") -> ParticleCollection:
+    if not filename:
```

Comment:
I'm not sure if we should have this kind of hidden behaviour. Instead, probably better to keep the `load_pdg` function separate, and make it public. (Yes, I know it was me who implemented it like this...)

What do you think @spflueger?

Reply:
Agreed. Then its more clear from the code whats happening